docs/mobile-apps/automated-testing/appium.md

```diff
@@ -1,437 +1,165 @@
 
-## Appium Overview
+You can see more detailed and comprehensive instructions for all use cases can be found in the [Real Devices](https://docs.saucelabs.com/mobile-apps/automated-testing/appium/real-devices) and [Virtual Devices](https://docs.saucelabs.com/mobile-apps/automated-testing/appium/virtual-devices) docs.
 
-* Originally developed by Dan Cueller as a way to take advantage of the UIAutomation framework for Apple iOS to run tests against native mobile applications.
-* Most commonly used for testing native and hybrid mobile applications for iOS and Android.
-* You can also leverage it to test websites on a mobile device.
-* Uses the same syntax as [Selenium](https://www.selenium.dev).
-* Shares similarities with Selenium's ability to automate interaction with a website through a mobile browser.
 
 ## What You'll Need
 
-You can read about Appium's system requirements [here](http://appium.io/).
+* [Sample Tests](https://github.com/saucelabs-training/quickstart-appium-java)
+* The [Swag Labs Sample App](https://github.com/saucelabs/sample-app-mobile/releases) (App Tests)
+* A [Sauce Labs Username and Access Key](https://saucelabs.com/sign-up)
+* An IDE such as [IntelliJ IDEA](https://www.jetbrains.com/idea/download/#section=mac)
+* [Maven](https://maven.apache.org/install.html)
 
-## Appium Architecture
+## Quickstart – Test a Mobile Web Browser on Sauce Labs VMs
 
-Appium has a client-server architecture.
+### Step 1: Install Dependencies
+Make sure you [have Java, an IDE, and Maven installed](https://training.saucelabs.com/codelabs/Module1-SeleniumJava/index.html?index=..%2F..SeleniumJava#4) on your computer. To test a mobile web browser you will need either Android Studio or XCode to run an Emulator/ Simulator for testing on.
 
-### Appium Client
+You will also want to set your `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY` as [environment variables](https://www.youtube.com/watch?v=3K1Eu0eTha8).
 
-The Appium client includes a [set of client libraries](http://appium.io/downloads) for various scripting languages in which you write your test scripts based on the Selenium WebDriver API.
+### Step 2: Get Test Code
+In the [Sauce Appium sample repo](https://github.com/saucelabs-training/quickstart-appium-java/tree/main/src/test/java/tests), you will be running the test in the `/src/test/java/tests` directory called `AppiumAndroidEmuWebTest.java`
 
-### Appium Server
+With TestNG you specify the tests you want to run using an .xml file in the`/resources/xml` [directory](https://github.com/saucelabs-training/quickstart-appium-java/tree/main/src/test/resources/config), by editing the `pom.xml` file. Update the `testngXmlFile` tag to use the correct TestNG xml file:
```

Comment:
1. When you are writing a step-by-step procedure, each step should begin with an action rather than an explanation. e.g., "Edit pom.xml in [this location] to reference the sample test found [here]."
2. The reference is `/resources/xml`, but the link it to `resources/config` (there is no `/xml` folder in the `resources` directory in the link you provide)? 
3. `pom.xml` is not located in the directory you provide; it is in the root folder of the repo. (https://github.com/saucelabs-training/quickstart-appium-java/blob/main/pom.xml)

Reply:
Updated this to two 
1) Open and view your test code
2) Open and edit pom.xml